README.md

```diff
@@ -21,30 +21,33 @@ It's based on:
 
 ## Required software
 
-FsAutoComplete can run on .NET/mono or .NET Core.
+FsAutoComplete can run on .NET/mono or .NET Core. We highly recommend using .Net Core version of FSAC as we plan to remove "full framework" version soon - [read more](https://github.com/fsharp/FsAutoComplete/issues/506)
 
-### FSAC .NET
+### FSAC .NET Core
 
-* on windows: [Microsoft Build Tools 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48159)
-* on unix/mac: Required: Mono >= 5.12, Recommended: Mono >= 5.18
+* .NET Core Sdk (Required: >= 2.1, Recommended: >= 3.0 )
```

Comment:
The use of .net core sdk tool requires 3.0+, doesn't it? And the global.json value will require it to be 3.1.x, so we should note that here.

Reply:
This is requirement for running FSAC, not for building. Building section is below.